app/forms/hyrax/composition_form.rb

```diff
@@ -4,23 +4,22 @@ module Hyrax
   # Generated form for Composition
   class CompositionForm < Hyrax::Forms::WorkForm
     self.model_class = ::Composition
-    self.terms -= [:creator, :keyword, :subject]
-    self.terms += [:instruments, :date, :tags,
-                   :length, :genre, :software, :medium, :composer]
+    self.terms += [:duration, :medium, :resource_type,
+                   :technical_specs, :program_notes]
 
-    self.required_fields = [:title, :instruments]
+    self.required_fields = [:title, :subject]
+
+    # composer, title, instruments, description, tags,
+    # genre, date_created, duration, score, program_notes,
+    # technical specs  
+    def primary_terms
+      [:creator, :title, :subject, :description, :keyword, 
+       :resource_type, :date_created, :duration, :medium,
+       :source, :technical_specs, :program_notes]
```

Comment:
Question: Should `:source` be a field for the score? Or, should we consider having the score be an uploaded file?

Reply:
In the requirement, it says it is a field with a link to a score file

Reply:
Oh okay. Then, for this PR, it makes sense the way it is. 

Even still, I wonder if, eventually, it should be an uploaded file. It'll get thumbnails generated, and the author could use it as a thumbnail to represent a composition. Also, if we decide to enable analytics, then users can potentially keep track of now many downloads it's had.